pkg/controller/signup.go

```diff
@@ -31,6 +31,18 @@ func (s *Signup) PostHandler(ctx *gin.Context) {
 	ctx.Writer.Header().Set("Content-Type", "application/json")
 	// TODO call s.signupService.CreateUserSignup() to create the actual resource in Kube API Server
 	ctx.Writer.WriteHeader(http.StatusOK)
+
+	userSignup, err := s.signupService.CreateUserSignup(ctx.GetString(context.UsernameKey), ctx.GetString(context.SubKey))
+	if err != nil {
+		ctx.JSON(http.StatusInternalServerError, gin.H{
```

Comment:
We should use `errors.AbortWithError()` instead.

Reply:
Addressed in https://github.com/codeready-toolchain/registration-service/pull/49